--- node_modules/qunit/qunit/qunit.orig.js	2019-04-13 13:02:25.000000000 +0700
+++ node_modules/qunit/qunit/qunit.js	2019-04-20 12:10:07.000000000 +0700
@@ -3996,7 +3996,7 @@
   					expected = String(expected);
 
   					// Expected is a constructor, maybe an Error constructor
-  				} else if (expectedType === "function" && actual instanceof expected) {
+  				} else if (expectedType === "function" && expected.prototype !== undefined && actual instanceof expected) {
   					result = true;
 
   					// Expected is an Error object
